source/02-combat-prototype/actors/physics/PlatformPhysics.gd

```diff
@@ -1,25 +1,23 @@
 class_name PlatformPhysics
 extends Node
 
-signal started
-signal finished
-
-export (NodePath) var platform_actor_path = ".."
-onready var platform_actor = get_node(platform_actor_path)
+export (NodePath) var actor_path = ".."
+onready var actor = get_node(actor_path)
 
 export (bool) var enabled = false setget set_enabled
 
 func _ready():
-	set_physics_process(false)
+	set_physics_process(enabled)
 
 
-func apply(): #virtual method
+func apply():
 	pass
 
 
-func stop(): #virtual method
+func stop():
 	pass
 
 
 func set_enabled(enable):
 	enabled = enable
+	set_physics_process(enabled)
```

Comment:
Since this is happening in the parent class I think we can remove these lines in `Gravity`
https://github.com/pigdevstudio/kitchen-tales/blob/2f10b8d75a69ef0044496e5de45b4ba76a856073/source/02-combat-prototype/actors/physics/Gravity.gd#L24